scripts/pre-build.js

```diff
@@ -84,6 +82,39 @@ const start = async (source) => {
 
   console.log('Updating sidebar.js');
   await createSidebar('docs', path.join(process.cwd(), 'sidebars.js'));
+
+  console.log('Downloading translations');
+  const locales = await downloadTranslations(I18N_FOLDER);
+
+  for (const locale of locales) {
+    const localeDocs = path.join(
+      I18N_FOLDER,
+      locale,
+      'docusaurus-plugin-content-docs',
+      'current'
+    );
+    const staticResources = ['fiddles', 'images'];
+
+    console.log(`Copying static assets to ${locale}`);
+    for (const staticResource of staticResources) {
+      await fs.copy(
+        path.join(DOCS_FOLDER, staticResource),
+        path.join(localeDocs, 'latest', staticResource)
```

Comment:
Exactly!

A bit more context for anyone interested. The folder `current` is a Docusaurus convention to point to the latest version as you figured out (so `/docs/`). Because we don't want collision with the current documentation (www.electronjs.org/docs) and we want to have versioned docs sometime in the future, we append `/latest` so the "current" documentation is available via www.electronjs.org/docs/latest.

Older versions will be available via www.electronjs.org/docs/v12 (or a similar path yet TBD)

I haven't dig how the versioning of docs works exactly with docusaurus but its very likely it will be under another folder at the same level as `/current` like `/v12`.